### PR TITLE
Add copyright and license link to TileLayer example

### LINKED ISSUE #5993: OpenStreetMap attribution in TileLayer example
Dear maintainers,

#### Link
https://deck.gl/examples/tile-layer/

#### Description
In the tile-layer example, osm-carto tiles are used to illustrate how we can use tiles in Deck.gl with TileLayer help.
This example doesn't show proper attribution for OSM tiles, which prevent users to understand how they can properly credit such tile sets.
See https://www.openstreetmap.org/copyright

Is it possible to complete this example with appropriate attribution please ?

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
Closes #5993

### CODE REVIEW

File: examples/website/map-tile/app.js [hunk 1]
```diff
@@ -15,6 +15,21 @@ const INITIAL_VIEW_STATE = {
   bearing: 0
 };
 
+const COPYRIGHT_LICENCE_STYLE = {
```
Comment (kylebarron) [inline examples/website/map-tile/app.js:18]: Should we standardize on American English? `license`

File: examples/website/map-tile/app.js [hunk 2]
```diff
@@ -68,14 +83,26 @@ export default function App({showBorder = false, onTilesLoad = null}) {
     }
   });
 
+  const renderCopyrightAndLicense = () => (
+    <div style={COPYRIGHT_LICENCE_STYLE}>
+      {'© '}
+      <a style={LINK_STYLE} href="http://www.openstreetmap.org/copyright" target="blank">
+        OpenStreetMap
+      </a>
+      {' contributors'}
```
Comment (kylebarron) [inline examples/website/map-tile/app.js:92]: I think the whole text should be a link, not just the word `OpenStreetMap`

Comment (kylebarron) [reply examples/website/map-tile/app.js:92]: On OSM's [own map](https://www.openstreetmap.org/#map=5/38.007/-95.844) the hyperlink includes `contributors` but not the copyright symbol 🤷‍♂️ 
image

**Review by kylebarron — APPROVED** [review]

File: examples/website/map-tile/app.js [hunk 3]
```diff
@@ -68,14 +83,26 @@ export default function App({showBorder = false, onTilesLoad = null}) {
     }
   });
 
+  const renderCopyrightAndLicense = () => (
```
Comment (Pessimistress) [inline examples/website/map-tile/app.js:86]: Move this out of the function

File: examples/website/map-tile/app.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/website/map-tile/app.js]: Fixed.

File: examples/website/map-tile/app.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/website/map-tile/app.js]: Done

File: examples/website/map-tile/app.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply examples/website/map-tile/app.js]: Done

**Pull request merged**